packages/client/hooks/useGetRepoContributions.ts

```diff
@@ -0,0 +1,90 @@
+import graphql from 'babel-plugin-relay/macro'
+import {isNotNull} from 'parabol-client/utils/predicates'
+import {useMemo} from 'react'
+import {useFragment} from 'react-relay'
+import {useGetRepoContributions_teamMember$key} from './../__generated__/useGetRepoContributions_teamMember.graphql'
+
+export type Repo = {
+  readonly id: string
+  readonly nameWithOwner: string
+}
+
+const useGetRepoContributions = (teamMemberRef: useGetRepoContributions_teamMember$key): Repo[] => {
```

Comment:
Previously, in the filter menu, we were getting repos by contributions, but in the new issue menu, we were querying them by organization. By moving this to a hook and querying the same fields, we save an additional query if the user opens the filter menu before the new issue menu, for example.